Grievance_Backend/grievance/src/main/java/com/grievance/controller/DepartmentController.java

```diff
@@ -57,14 +58,18 @@ public class DepartmentController {
 //  }
   @PostMapping("/save")
   public ResponseEntity<?> saveDepartment(
-    @RequestParam final String email,
-    @RequestParam final String password,
+    @RequestHeader final String email,
+    @RequestHeader final String password,
     @RequestBody final DepartmentInDto departmentInDto
   ) {
       Boolean boolean1 = authenticatingUser.checkIfUserIsAdmin(email, password);
```

Comment:
give some meaningful name instead of boolean1 (what is this variable doing).

Reply:
okay, will change it to isAdmin as I'm checking is the user is admin here.